### PR TITLE
Badge Profile: add "follow all awardees"

### PR DESCRIPTION
**Pull request by WShakesp1**
Add follow all awardees:

image
image
image

### CODE REVIEW

File: src/lib/BadgeProfile.js [hunk 1]
```diff
@@ -200,11 +202,67 @@ export default function BadgeProfile({ ev, ...rest }) {
...
   const image = findTag(ev.tags, "image");
   const naddr = encodeNaddr(ev);
   //const thumb = findTag(ev.tags, "thumb");
+
+  async function followPubKeys(awardEvents) {
+    const followEvent = {
+      kind: CONTACT_LIST,
+      tags: [],
+      created_at: dateToUnix(),
+      content: "",
```
Comment (verbiricha) [inline src/lib/BadgeProfile.js:224]: Can we keep the content from the old event? in practice clients are putting a JSON string with relay metadata there and don't want to nuke it.

Comment (WShakesp1) [reply src/lib/BadgeProfile.js:224]: makes sense, i'll revert

Comment (WShakesp1) [reply src/lib/BadgeProfile.js]: done

**Comment by verbiricha** [comment]
Thanks @WShakesp1 Merged and released.

**Pull request merged**